compiler-rt/lib/sanitizer_common/sanitizer_common_libcdep.cpp

```diff
@@ -219,6 +219,32 @@ static void StopStackDepotBackgroundThread() {
 static void StopStackDepotBackgroundThread() {}
 #endif
 
+void MemCpyAccessible(void *dest, const void *src, uptr n) {
+  if (TryMemCpy(dest, src, n))
```

Comment:
I am not fully sure about this optimization. In the case where there are holes this makes it more expensive. Does the caller generally know that?

Reply:
Not sure as well. On some examples fast pass 4x faster than page by page.